archiver.go

```diff
@@ -160,7 +160,15 @@ func selectEndpoint(endpoints []string) (string, error) {
 		return "", ErrEndpointsEmpty.New()
 	}
 
-	// TODO check which endpoint to use
+	var endpointsOrder = []string{"git://", "https://", "http://"}
```

Comment:
@smola but do not instance `endpointsOrder` on every selectEndpoint call! put it outside.